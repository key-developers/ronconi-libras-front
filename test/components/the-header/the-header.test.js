import TheHeader from '@/components/the-header/the-header'
import { mount } from '@vue/test-utils'

describe('Components > TheHeader', () => {
  const wrapper = mount(TheHeader)

  it('matches snapshot', () => {
    expect(wrapper.element).toMatchSnapshot()
  })
})
