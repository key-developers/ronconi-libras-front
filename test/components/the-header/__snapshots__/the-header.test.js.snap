// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components > TheHeader matches snapshot 1`] = `
<div
  class="header"
>
  
  Home

</div>
`;
